models/Recipe.model.js

```diff
@@ -1,10 +1,52 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: {
+    type: String,
+    unique: true,
+    require: true,
+  },
+  level: {
+    type: String,
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+  },
+  ingredients: {
+    type: [String],
+  },
+  cuisine: {
+    type: String,
+    require: true,
+  },
+  dishType: {
+    type: String,
+    enum: [
+      "breakfast",
+      "main_course",
+      "soup",
+      "snack",
+      "drink",
+      "dessert",
+      "other",
+    ],
+  },
+  duration: {
+    type: Number,
+    min: 0,
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg",
+  },
+  creator: {
+    type: String,
+  },
+  created: {
+    type: Date,
+    default: "2023-02-09",
```

Comment:
Here they ask you to pass today as the default date. There is a method in js that always give you the current day `Date.now()`:

```
  created: {
    type: Date,
    default: Date.now
  }
```